ambuda/views/proofing/project.py

```diff
@@ -391,216 +413,98 @@ def replace(slug):
     if project_ is None:
         abort(404)
 
-    form = ReplaceForm(request.form)
+    form = ReplaceForm(request.args)
```

Comment:
Is `request.args` your preferred way to pass params to the server? After experimenting a bit, I came out in favor of form. At some point args will break as the `query` or `replace` string grows in size or perhaps has sensitive info or has data unsupported in the url. Perhaps I'm missing your reasoning here.

Reply:
You see 

> Thanks for the review!
> 
> > For instance, I do not simply run a "search-replace" all in an editor when renaming a variable. Instead, I go through each line to ensure precision.
> 
> I agree that a blind search/replace is unwise and that the user must review each change to ensure quality. My feeling is that the phase where the user checks/unchecks each item in the result set accomplishes this purpose already. I'm curious why you feel differently.

In my opinion, it's easy to verify the checks/unchecks when there are only a handful of matches on a single search+replace page. However, as the number of matches increases (for example, hundreds of matches), the pagination will spread the checks/unchecks across multiple search+replace pages. This can make it difficult to keep track of which checks/unchecks have already been applied, leading to potential tracking issues.